assets/wire/utils.js

```diff
@@ -205,7 +205,7 @@ export function shortHighlightedtext(html, maxLength = 40) {
   
     while (node && count < maxLength) {
         const content = node.textContent.trim();
-        const words = content.split(/\s/).filter(w => w);
```

Comment:
Why change from split by `whitespace` to split by `followed by [A-Z]`?

Reply:
actually that split not give us desired output in the case if search term is found in the last line of the paragraph and paragraph is too long.

Reply:
It would be good to have some unit tests for this

Reply:
sure